package.json

```diff
@@ -16,7 +16,8 @@
   },
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "(cd ./src; ./publish.sh) && npm run test:size",
```

Comment:
Could you change it to `build`? The command doesn't run any browser-based tests. Thus the name `test` could be misleading.